docs/contributing/versioning.md

```diff
@@ -41,32 +41,6 @@ This is to ensure the community can get involved with the decision.
 
 Make an active effort to involve the community, this might be in the form of presentations or meetings.
 
-## Deprecation
```

Comment:
There is an anchor link to this section [further up in the file](https://github.com/alphagov/govuk-frontend/pull/2679/files#diff-86a4fd213ed466a2707fbf35c759f4bf8fdf9a24139864757994ef8bc22dcdb1R34) which will break now the section has been removed.

Reply:
I've resolved this by removing the migration section (which didn't really add anything over what was in the managing change doc) and updating the link to point to the managing change doc instead.